server/validations/validateRequest.js

```diff
@@ -0,0 +1,54 @@
+import validator from 'validator/';
+import isEmpty from 'lodash/isEmpty';
+
+class ValidateRequest {
+  /**
+   * @description Validates create request inputs
+   *
+   * @param {Object} req
+   * @param {Object} res
+   * @param {Function} next
+   *
+   * @return {Function} next
+   */
+  static create(req, res, next) {
+    const { body } = req;
+    const errors = {};
+
+    if (!body.type || validator.isEmpty(body.type.trim())) { errors.type = 'type is required'; }
+
+    if (body.type &&
+       !validator.isIn(body.type.trim().toLowerCase(), ['repairs', 'maintenance'])) {
+      errors.type = 'type must be either of: repairs or maintenance';
+    }
+
+    if (!body.category || validator.isEmpty(body.category.trim())) { errors.category = 'category is required'; }
+
+    if (body.category &&
+       !validator.isIn(body.category.trim().toLowerCase(), ['computers', 'electronics', 'furnitures'])) {
+      errors.category = 'category must be either of: computers, electronics, furnitures';
+    }
+
+    if (!body.item || validator.isEmpty(body.item.trim())) { errors.item = 'item is required'; }
```

Comment:
Make sure you configure Eslint, to check for a maximum of 80 characters per line. This code seems non-uniform.

Reply:
I've fixed the line length and indentation